### ISSUE TITLE
Typescript + Reactotron + Flipper

### ISSUE THREAD

**Issue by StagasaurusRex** [infinitered/reactotron-react-native#492]
I am getting an error trying to add Reactotron + Flipper w/ typescript.

> Could not find a declaration file for module 'reactotron-react-native/dist/flipper'.

<img width="642" alt="image" src="https://user-images.githubusercontent.com/4968936/89739974-cf629180-da4a-11ea-9987-8d96b48cdebf.png">

It seems like types are available in reactotron-react-native/dist/types/FlipperConnectionManager, but they're not found.

Is there any easy fix on my side or is this something the library needs to fix?

**Comment by pcarollo84** [comment]
I have the same issue. Did you find a way to fix it @StagasaurusRex ? 
With this configuration I can't make it work on Flipper as plugin. It only works with the standalone app.

**Comment by StagasaurusRex** [comment]
Not really. I just added `// @ts-ignore` before that line. That causes an issue with my linter, but at least it works
